index/style.css

```diff
@@ -1,57 +1,89 @@
 *{
-  margin: 0;
+  margin-top: 0;
+}
+
+html{
+  /*css tricks had a way to adjust an img in a page at all times, no matter screen or browser size*/
+  background: url("background.") no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
these vendor prefixes are becoming obsolete (thank god).  It is good to know about for older browser support, but you should be able to use background-size across all browsers.  if you are ever curious, there are sites that will show what browsers support specific functions.  here is an example: http://caniuse.com/#search=background-size